application/src/main/java/net/corda/v5/application/interop/parameters/TypeQualifier.java

```diff
@@ -0,0 +1,68 @@
+package net.corda.v5.application.interop.parameters;
+
+import org.jetbrains.annotations.NotNull;
+
+import java.util.List;
+import java.util.Objects;
+
+public class TypeQualifier {
+    @NotNull
+    private final String owner;
```

Comment:
Once again what does Owner mean?

Reply:
Right from a comment i can see owner is the publisher for the facade like org.r3.com 
I would suggest we change the name to publisher rather than owner. 
@r3domfox

Reply:
Will also include the javadoc for this